api/src/main/java/io/strimzi/api/kafka/model/mirrormaker2/KafkaMirrorMaker2Spec.java

```diff
@@ -21,17 +23,22 @@
         builderPackage = Constants.FABRIC8_KUBERNETES_API
 )
 @JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({"version", "replicas", "image", "connectCluster", "clusters", "mirrors", "resources", "livenessProbe",
-    "readinessProbe", "jvmOptions", "jmxOptions", "logging", "clientRackInitImage", "rack", "metricsConfig", "tracing",
-    "template", "externalConfiguration" })
+@JsonPropertyOrder({"version", "replicas", "image", "connectCluster", "clusters", "target", "mirrors", "resources",
+    "livenessProbe", "readinessProbe", "jvmOptions", "jmxOptions", "logging", "clientRackInitImage", "rack",
+    "metricsConfig", "tracing", "template", "externalConfiguration" })
 @EqualsAndHashCode(callSuper = true, doNotUseGetters = true)
 @ToString(callSuper = true)
 public class KafkaMirrorMaker2Spec extends AbstractKafkaConnectSpec {
     private List<KafkaMirrorMaker2ClusterSpec> clusters;
     private String connectCluster;
+    private KafkaMirrorMaker2TargetClusterSpec target;
     private List<KafkaMirrorMaker2MirrorSpec> mirrors;
 
     @Description("Kafka clusters for mirroring.")
+    @PresentInVersions("v1beta2")
+    @Deprecated
+    @DeprecatedProperty(description = "The `clusters` section is deprecated and will be removed in the `v1` CRD API. " +
+            "Please use the `.spec.target` and `.spec.mirrors.source` sections instead.")
```

Comment:
```suggestion
            "Please use the `.spec.target` and `.spec.mirrors[].source` sections instead.")
```

At least this is how we are referring to the mirrors elsewhere (i.e. CHANGELOG)